MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -183,14 +151,12 @@ private IPlaybackManager CurrentPlaybackManager
         /// <returns></returns>
         public async Task Play(IEnumerable<IMediaFile> mediaFiles)
         {
-            var enumerable = mediaFiles as IList<IMediaFile> ?? mediaFiles.ToList();
             MediaQueue.Clear();
-            MediaQueue.AddRange(enumerable);
+            MediaQueue.AddRange(mediaFiles);
 
-            await Task.WhenAll(
-                PlayNext(),
-                GetMediaInformation(enumerable),
```

Comment:
@martijn00 Do you know why the `GetMediaInformation()` was called here. If i'm not mistaken this was called anyway for each media file individually in the `PlayNext()` method, I therefore removed the call here and assumed that it would be enough just to call it in `PlayNext()`.
https://github.com/martijn00/XamarinMediaManager/pull/95/files#diff-c67432e6607274e8fcd1823fd67513e9L91